cli/src/test/java/org/jboss/as/cli/parsing/test/CommandTestCase.java

```diff
@@ -264,6 +264,22 @@ public void testQuotesInQuotes() throws Exception {
         assertEquals("\"a \\\"b\\\"\"", cmd.getPropertyValue("--arg"));
     }
 
+    @Test
+    public void testCurlyBrackets() throws Exception {
+        DefaultCallbackHandler cmd = parse("cmd --arg={a ");
+        assertEquals("cmd", cmd.getOperationName());
+        assertEquals("--arg", cmd.getLastParsedPropertyName());
+        assertEquals("{a", cmd.getPropertyValue("--arg"));
```

Comment:
I'm wondering whether the expected value should include the space too, i.e '{a ', since the value was not complete.